src/System.Runtime/src/System/Runtime/CompilerServices/CompilationRelaxations.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.CompilerServices 
+{
+    [Flags]
+    [System.Runtime.InteropServices.ComVisible(true)]
+    public enum CompilationRelaxations : int
```

Comment:
All of these are already exposed (as public) in S.P.CoreLib. I was hoping some of them can be pulled out from it into S.R. 

Should I take out CompilationRelaxationsAttribute from CoreLib or should I leave all of them inside CoreLib? What do you think?